### PR TITLE
osd 4489 app-sre integration

### PR DESCRIPTION
**Pull request by amolgautam25**
app-sre pipeline  configuration

### CODE REVIEW

File: config/config.go [hunk 1]
```diff
@@ -0,0 +1,6 @@
+package config
+
+const (
+	OperatorName      string = "aws-account-shredder"
```
Comment (fahlmant) [inline config/config.go:4]: Do we need these? This is not an operator

File: project.mk [hunk 2]
```diff
@@ -0,0 +1,9 @@
+OPERATOR_NAME?=$(shell sed -n 's/.*OperatorName .*"\([^"]*\)".*/\1/p' config/config.go)
```
Comment (fahlmant) [inline project.mk:1]: Same thing here. Not an operator so do we need these?

Comment (amolgautam25) [reply project.mk]: changes the variables

File: config/config.go (hunk 1 above)
Comment (amolgautam25) [reply config/config.go]: changed the variables.

File: config/config.go [hunk 3]
```diff
@@ -1,6 +1,6 @@
 package config
 
 const (
-	OperatorName      string = "aws-account-shredder"
-	OperatorNamespace string = "aws-account-shredder"
+	AwsClientName      string = "aws-account-shredder"
```
Comment (fahlmant) [inline config/config.go:4]: `ApplicationName`?

File: standard.mk [hunk 4]
```diff
@@ -18,12 +18,11 @@ COMMIT_NUMBER=$(shell git rev-list `git rev-list --parents HEAD | egrep "^[a-f0-
 CURRENT_COMMIT=$(shell git rev-parse --short=8 HEAD)
 OPERATOR_VERSION=$(VERSION_MAJOR).$(VERSION_MINOR).$(COMMIT_NUMBER)-$(CURRENT_COMMIT)
```
Comment (fahlmant) [inline standard.mk:19]: Change to `APP_VERSION`

File: standard.mk [hunk 5]
```diff
@@ -18,12 +18,11 @@ COMMIT_NUMBER=$(shell git rev-list `git rev-list --parents HEAD | egrep "^[a-f0-
...
 
-IMG?=$(IMAGE_REGISTRY)/$(IMAGE_REPOSITORY)/$(IMAGE_NAME):v$(OPERATOR_VERSION)
+IMG?=$(IMAGE_REGISTRY)/$(IMAGE_REPOSITORY)/$(IMAGE_NAME):latest
```
Comment (fahlmant) [inline standard.mk:21]: Let's still tag this using `APP_VERSION`

Comment (amolgautam25) [reply standard.mk]: resolved.

File: Makefile [hunk 6]
```diff
@@ -3,6 +3,13 @@ SHELL := /usr/bin/env bash
 OPERATOR_DOCKERFILE = ./deploy/Dockerfile
```
Comment (fahlmant) [inline Makefile:3]: Is this used anywhere?

**Pull request merged**